TA-puppet-alert-actions/bin/ta_puppet_alert_actions/modalert_puppet_run_plan_helper.py

```diff
@@ -0,0 +1,242 @@
+
+# encoding = utf-8
```

Comment:
probably wont hurt to stay to be explicit, but UTF-8 is the default encoding within py3
https://peps.python.org/pep-3120/

Reply:
Since it works without it and py2 is deprecated, let's go ahead and remove it. 👍